.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "restructuredtext.confPath": ""
+}
```

Comment:
Could you remove this? You can add to `.gitignore` if you want, also. I think we don't need it in this repository

Reply:
Updated `.gitignore`

Reply:
Not seeing the gitignore update

Reply:
Here it is https://github.com/django-ses/django-ses/pull/234/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R10

Reply:
I see, the file is still checked in though, could you push a commit to remove this file?